io.github.purplehorrorrus.Meridius.yml

```diff
@@ -0,0 +1,38 @@
+app-id: io.github.purplehorrorrus.Meridius
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+base: org.electronjs.Electron2.BaseApp
+base-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: meridius
+separate-locales: false
+modules:
+  - name: meridius
+    buildsystem: simple
+    build-commands:
+      - cp -a * /app/bin
+      - install -Dm644 $FLATPAK_ID.desktop /app/share/applications/$FLATPAK_ID.desktop 
+      - install -Dm644 $FLATPAK_ID.metainfo.xml /app/share/metainfo/$FLATPAT_ID.metainfo.xml
+      - install -Dm644 build/icons/256x256.png /app/share/icons/hicolor/256x256/apps/$FLATPAK_ID.png
+    sources:
+      - type: archive
+        url: https://github.com/PurpleHorrorRus/Meridius/releases/download/v2.4.11/meridius-2.4.11.tar.gz
+        sha256: 0f3028ed4fa2f04ad7f5f455da8ebc5567feaa92db8c9760dd7eb5637d21aa16
+      - type: script
```

Comment:
this script is never installed.

Reply:
And how can I run app without this script?

Reply:
how can you run the script if it's not in the package?

Reply:
With a similar command specified in the script

Reply:
I'll rephrase. This is never installed, so it is never part of the package. I'm not questioning why there is a script, just noting that it is not installed, because you never install it.

So it won't work.

Reply:
I don't understand this. Application successfully extracted and installed with ```flatpak-builder``` and ```flatpak install```. What do I need to change?

Reply:
- install -D meridius /app/bin/meridius